packages/api-explorer/src/components/SideNav/SideNavMethods.tsx

```diff
@@ -92,26 +94,56 @@ const SideNavMethodsLayout: FC<MethodsProps> = ({
                   {highlightHTML(pattern, method.summary)}
                 </Heading>
               </li>
-            </NavLink>
+            </SideNavLink>
```

Comment:
For future consideration: Might be interesting to look at leveraging `ListItem` from `@looker/components` here instead.